tests/unit/conftest.py

```diff
@@ -0,0 +1,14 @@
+from typing import Generator
+from unittest.mock import Mock
+
+import pytest
+from great_expectations.expectations import Expectation
+from pytest_mock import MockerFixture
+
+
+@pytest.fixture
+def mock_gx(mocker: MockerFixture) -> Generator[Mock, None, None]:
```

Comment:
This should have a docstring. If we hadn't talked about this before I'd be a bit lost.